install/linux/usr/share/odemis/sim/cryosecom-sim.yaml

```diff
@@ -231,8 +232,9 @@ CryoSECOM: {
     POS_ACTIVE_RANGE: {'x':  [0.00, 0.003], 'y':  [-0.003, -0.001], 'z':  [0.00, 0.003], } , # stage Z – lens Z, when the lens Z is the closest (highest) from the stage
     FAV_POS_ACTIVE: {'rx': 0.00, 'rz': 0.0, 'x':  0.0, 'y':  -0.002, 'z':  0.002},
     FAV_POS_DEACTIVE:  {'rx': 0.00, 'rz': 0.0, 'x':  0.01, 'y':  0.00, 'z':  -0.001},
+    FAV_AREA:  [0, 0, 0.002, -0.002]  # left, top, right, bottom
```

Comment:
Please change to `[0, -0.002, 0.002, 0]`... or wouldn't it make more sense for the simulator to have something centered around 0? => `[-0.001, -0.001, 0.001, 0.001]` ?